modules/rubiconBidAdapter.js

```diff
@@ -15,10 +15,12 @@ import {
 } from '../src/utils.js';
 import {registerBidder} from '../src/adapters/bidderFactory.js';
 import {config} from '../src/config.js';
-import {BANNER, VIDEO} from '../src/mediaTypes.js';
+import {BANNER, VIDEO, NATIVE} from '../src/mediaTypes.js';
 import {find} from '../src/polyfill.js';
 import {Renderer} from '../src/Renderer.js';
 import {getGlobal} from '../src/prebidGlobal.js';
+import {ortbConverter} from '../libraries/ortbConverter/converter.js';
+import {pbsExtensions} from '../libraries/pbsExtensions/pbsExtensions.js';
 
 const DEFAULT_INTEGRATION = 'pbjs_lite';
```

Comment:
With this example mediaTypes object I expect to see `skippable: true` passed to PBS but it does not.

Maybe it is something wrong with the default processor?

```
{
    "api": [
        2
    ],
    "pos": 1,
    "mimes": [
        "video/mp4",
        "application/x-mpegURL",
        "video/MP2T",
        "video/3gpp",
        "video/quicktime",
        "video/x-msvideo",
        "video/x-ms-wmv",
        "application/javascript",
        "video/ogg"
    ],
    "context": "instream",
    "skipmin": 30,
    "linearity": 1,
    "protocols": [
        2,
        3,
        5,
        6,
        7,
        8,
        11,
        12,
        13,
        14
    ],
    "skipafter": 5,
    "skippable": true,
    "playerSize": [
        [
            640,
            480
        ]
    ],
    "maxduration": 30,
    "minduration": 1,
    "playbackmethod": [
        2
    ]
}
```

Reply:
`skippable` doesn't exist in openrtb 2.5 specs, neither in pbjs docs, and in our previous code. Probably the property that makes sense is `skip` here

Reply:
well alrighty then. Not sure how I was getting it sent to PBS in my testing of a pub page then. Oh well